app/src/main/java/chat/rocket/android/chatroom/adapter/ChatRoomAdapter.kt

```diff
@@ -119,6 +138,23 @@ class ChatRoomAdapter(
         }
     }
 
+
+    private fun shouldGroupMessage(a: Message, b: Message): Boolean {
+        a.sender?.let { u1 ->
+            b.sender?.let { u2 ->
+                if (u1.id.equals(u2.id)) {
```

Comment:
Is not it good to consider it when the alias is different?
RocketChat/Rocket.Chat.iOS#2684